config/all-in-one/templates/cp4waiops-custom.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         helm:
           valueFiles:
             - values.yaml
-          {{- if eq .Values.cp4waiops.profile "x-small" }}
+          {{- if eq .Values.cp4waiops.profile "x-small-custom" }}
             - values.x-small.yaml
```

Comment:
Maybe it's better to rename as `values.x-small-custom.yaml` so that it's easier to maintain (people do not have to remember which one has `-custom` suffix and which one does not)?

Reply:
Agree to rename as `values.x-small-custom.yaml` to match  profile `x-small-custom` .